### PR TITLE
240

### LINKED ISSUE #240: Make tournament length available to players
The Backstabber strategy currently has the number of turns hard coded as 200. I believe that in the original tournament the number of turns was available to the players.

### PR DESCRIPTION
**Pull request by uglyfruitcake**
Fixes #240 

Tournament length is now available to players as ```self.turns```

### CODE REVIEW

**Comment by meatballs** [comment]
Nice effort!

I think that we should make ```tournament.turns``` a property rather than a simple attribute. Your ```_create_player_list``` method would then become the setter for the property.

**Comment by marcharper** [comment]
Everyone happy with this? My latest PR includes a strategy that relies on the total number of rounds, and if this is how we will do it I'll make a quick change to that PR to use self.turns.

**Comment by drvinceknight** [comment]
Just looking through it now :)

File: axelrod/tournament.py [hunk 1]
```diff
@@ -25,10 +25,21 @@ def __init__(self, players, name='axelrod', game=None, turns=200,
...
 
+    @property
+    def players(self):
+        return self._players
+
+    @players.setter
+    def players(self, players):
+        newplayers = []
+        for player in players:
+            player.turns = self.turns
```
Comment (drvinceknight) [inline axelrod/tournament.py]: Small suggestion: call this `player.tournament_length` or something like that? `player.turns` could be understood as a counter of how many turns have happened so far or something like that?

Comment (drvinceknight) [inline axelrod/tournament.py]: A part from that small suggestion: this looks really great - great job.

**Comment by uglyfruitcake** [comment]
Tournament length is now available to players as self.tournament_length.

**Comment by drvinceknight** [comment]
:+1: (Waiting on travis, will merge asap)

**Pull request merged**